components/ImagePopover/index.js

```diff
@@ -0,0 +1,79 @@
+/** @flow */
+
+require('./styles.css');
+
+import React from 'react';
+var {PropTypes} = React;
+
+//todo: handle configuring desired placement with prop
+//todo: logic to ensure that edge-item popovers don't pop out of the viewport
+
+class ImagePopover extends React.Component {
+  onMouseEnter(e) {
+    e.preventDefault();
+    this.props.handleMouseEnter && this.props.handleMouseEnter(this.props.data)
+  }
+
+  onMouseLeave(e) {
+    e.preventDefault();
+    this.props.handleMouseLeave && this.props.handleMouseLeave(this.props.data)
+  }
+
+  onClick(e) {
+    e.preventDefault();
+    this.props.handleClick && this.props.handleClick(this.props.data)
+  }
+
+  getDirectionClass() {
+    if (this.props.direction) {
+      return prefix(direction);
```

Comment:
This was missed when I removed the prefix function.. I'll take it out an dadd it back when/if I implement passing in a direction